pkg/server/router/manifest.go

```diff
@@ -181,19 +180,20 @@ func (mr ManifestRouter) DeleteManifest(ctx context.Context, w http.ResponseWrit
 type SubmitApplicationRequest struct {
 	Application manifestsdk.CredentialApplication `json:"application" validate:"required"`
 	// Once we have JWT signed wrapper that can get the did this can be removed
```

Comment:
Is this comment still valid?

Reply:
it is, this will need to be changed to accept a verifiable presentation or jwt. will handle separately